src/cmds/versions/create.js

```diff
@@ -89,11 +114,11 @@ module.exports = class CreateVersionCommand {
       }),
       body: JSON.stringify({
         version,
-        codename: codename || '',
-        is_stable: main === 'true' || promptResponse.is_stable,
-        is_beta: beta === 'true' || promptResponse.is_beta,
-        from: fork || promptResponse.from,
-        is_hidden: promptResponse.is_stable ? false : !(isPublic === 'true' || promptResponse.is_hidden),
+        codename: opts.codename || '',
+        is_stable: opts.main || answers.main,
+        is_beta: opts.beta || answers.beta,
+        from: opts.fork || answers.fork,
+        is_hidden: opts.main || answers.main ? false : !('public' in opts || answers.public),
```

Comment:
Nit (and not your fault) but it might be nice to extract these sub-expressions into a separate variables so this is a bit more readable:
```suggestion
        is_hidden: isMain ? false : !isPublic,
```